src/main/java/uk/gov/register/resources/RecordResource.java

```diff
@@ -20,18 +20,18 @@
 @Path("/")
 public class RecordResource {
     private final HttpServletResponseAdapter httpServletResponseAdapter;
+    private final RequestContext requestContext;
     private final RegisterReadOnly register;
     private final ViewFactory viewFactory;
     private final RegisterName registerPrimaryKey;
```

Comment:
`registerPrimaryKey` is not used any more, except in the constructor, can it be removed?